recipes/underworld/meta.yaml

```diff
@@ -0,0 +1,84 @@
+{% set name = "underworld" %}
+{% set version = "2.14.1" %}
+{% set sha256 = "ec922eb6d10d6a6031d5c76b106df0c79b122ee61c43900866f2017bf819b2eb" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/ad/44/e491e7faa66cd927b722304d3e0a7ecc5f6c24cefa6044868a84548303d5/underworld-{{ version }}b0.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [win or osx]
+  number: 0
+  script: PETSC_DIR=$PREFIX {{ PYTHON }} -m pip -vv install --no-deps .  
+  # shared objects to ignore when checking linkage information
+  missing_dso_whitelist:
+    - "**/site-packages/underworld/lib/**"
```

Comment:
Do you really want this? is there no way for you to compile a C library for this package, install it in the `lib` folder, then just use those shared libraries in python?

putting everything under `site-packages` is something you would do in `pypi` if you don't want to share shared objects. Sharing shared objects libraries is one of the advantages of conda.

Reply:
We don't need this. I'll remove this line.